M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -939,8 +1011,7 @@ projDim(Module) := opts -> (N1) -> (
 isGBDone := (myGB) -> (
     --a temporary function for finding out if a gb computation is done.
     myStr := status myGB;
-    --if (0 < #select("status: done", myStr)) then return true else return false;
-    return match("status: done", myStr);-- then  return true else return false;
+    if (0 < #select("status: done", myStr)) then return true else return false;
```

Comment:
Simplify this line to `0 < #select("status: done", myStr)`.

Reply:
Done, although I think the code is less readable now.

Reply:
Because of the missing return?  How do you like `return (0 < #select("status: done", myStr))`?

Reply:
Done.  Hopefully there will be a better way to get this information in the future anyways.  I'll push momentarily.